### DISCUSSION TITLE
Combustion

### DISCUSSION THREAD

**Discussion by AmirrezaHBSH** [firemodels/fds#13524] [Q&A] [answered]
Hello,

I have run a case with a pine needle bed inside the domain and an igniter placed beside the bed. Previously, the pyrolyzed gas was CH₄, which allowed combustion to occur and the fire to propagate along the bed. I then changed it to CO. However, now the flame generates at the leading edge of the bed but does not propagate.

Regardless of the heat of combustion of CH₄ and CO, as well as the igniter temperature, could you please advise on what parameter might be necessary to add or adjust to resolve this issue?

**Comment by drjfloyd** [comment, marked as answer]
If you are predicting fire spread you are doing research. Some questions you may want to consider:

Is CO a reasonable surrogate for wood pyrolyzates?
Have you correctly defined the REAC for CO?
Have you considered how the extinction model will interact with CO as the fuel?
How far away are you from propogation? Have you done any work at all to examine and compate the heat flux and heating of upstream needles between the two cases?

**Reply by AmirrezaHBSH** [reply]
Thank you. If I define a BC as an open boundary, is it necessary to define initial Air inside the domain?

**Reply by drjfloyd** [reply]
This is something you can answer yourself with a simple test case.
